tests/ui/pages/activity/activity.page.ts

```diff
@@ -1,12 +1,15 @@
 import Button from '../../elements/button';
+import Dropdown from '../../elements/dropdown';
 import List from '../../elements/list';
+import TextBox from '../../elements/text-box';
 import BasePage from '../base-page';
 
 const SELECTORS = {
   datasourceFilter: '[data-qa="datasource_filter"] >> ..',
   namespaceFilter: '[data-qa="namespace_filter"] >> ..',
   eventTypeSelect: '[data-qa="event_type_filter"] >> ..',
-  filterWithInput: filterName => `label:has-text("${filterName}") >> ..`,
+  filterTag: `label:has-text('Tag')>> ..`,
+  filterOwner: `label:has-text('Owner')>> ..`,
```

Comment:
Pls rename locators: 
```
filterTag -> tagFilter
filterOwner -> ownerFilter
```
`filterTag` and `filterOwner` will fit for method names. But not for selector names.